application/src/main/java/org/thingsboard/server/exception/ThingsboardErrorResponseHandler.java

```diff
@@ -184,6 +187,13 @@ private void handleRateLimitException(HttpServletResponse response, TbRateLimits
                         ThingsboardErrorCode.TOO_MANY_REQUESTS, HttpStatus.TOO_MANY_REQUESTS));
     }
 
+    private void handleMaxPayloadSizeExceededException(HttpServletResponse response, TbMaxPayloadSizeExceededException exception) throws IOException {
+        response.setStatus(HttpStatus.BAD_REQUEST.value());
```

Comment:
Http has a special status for large payload
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413